### PR TITLE
Smith 2015

### PR DESCRIPTION
**Pull request by SimonGreenhill**
I added a few more concepts,  but did not add "sit by fire after childbirth"

### CODE REVIEW

**Comment by xrotwang** [comment]
"sit by fire after childbirth" sounds like a candidate for parabank :)

File: concepticondata/concepticon.tsv [hunk 1]
```diff
@@ -2677,3 +2677,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 2676	TWO HUNDRED	Quantity	The cardinal number occurring after one hundred and ninety-nine and before two hundred one, represented in Roman numerals as CC and in Arabic numerals as 200.	Number	
 2677	SPEARTHROWER	Warfare and hunting	A spear-thrower or atlatl is a tool that uses leverage to achieve greater velocity in dart-throwing, and includes a bearing surface which allows the user to store energy during the throw.	Person/Thing	
 2678	HAIR OR FUR	The body	The cover of a mammal or a bird.	Person/Thing	
+2679	BLOW (WITH NOSE)	Basic actions and technology	To expel air through nose.	Action/Process	
+2670	MACHETE	Food and drink	A machete is a broad blade used either as an implement like an axe, or in combat like a short sword.	Person/Thing
```
Comment (LinguList) [inline concepticondata/concepticon.tsv]: ah, this is the problem: you started re-counting from 2679 to 2670, but need to count from 2680, etc. I had the same problem in the past ;-)

File: concepticondata/conceptrelations.tsv [hunk 2]
```diff
@@ -446,3 +446,6 @@ SOURCE	SOURCE_GLOSS	RELATION	TARGET	TARGET_GLOSS
 2678	HAIR OR FUR	narrower	580	FUR
 2678	HAIR OR FUR	narrower	1040	HAIR
 2381	FEATHER OR FUR OR HAIR	narrower	2678	HAIR OR FUR
+2672	UNCLE OR AUNT	narrower	1272	AUNT
```
Comment (LinguList) [inline concepticondata/conceptrelations.tsv]: sorry for bothering: we have the silent, no-where-stated convention to sort the "a or b" concepts alphabetically. This makes searching easier.

**Comment by SimonGreenhill** [comment]
Learn to count, Simon! these should be fixed now.

**Pull request merged**